.github/workflows/dashboard-build.yml

```diff
@@ -0,0 +1,57 @@
+name: Dashboard Build PR
+
+on:
+  push:
+    branches: [ master, develop ]
+
+defaults:
+  run:
+    working-directory: dashboard/src
+
+jobs:
+  build_dashboard:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+      - name: Compile Dashboard
+        run: |
+            # NB: We are in directory dashboard/src
+            yarn
+            yarn build
+            # If previous build folder exists and is same as new one, do not update.
+            # In any other case, we must update.
+            if ( [ -d "../build" ] && diff -qr build ../build ); then UPDATE_BUILD=0; else UPDATE_BUILD=1; fi
```

Comment:
Did you verify that two builds on the same source does not lead to any diff? For instance they do not contain timestamps  that may lead to a diff even though there are no differences in the source used?

Reply:
Yes, checked ! A same unchanged-code will generate the same build.